### DISCUSSION TITLE
Final Thinking Cap

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/21S-ASIAAM-191A#154] [Assignments]
### Note: Responses submitted after Wednesday, 6/2 at 2pm will not be eligible for awards.

What are your thoughts about the future of web development for civil society? Which visiting voice left the most impact on you? What connections (if any) would you like to maintain from this class going forward?

## Submission
Respond in 4-5 sentences before Thursday, 6/3 and respond to at least one other student.

**Comment by ghost** [comment]
All of the visiting voices were refreshing and eye-opening; however, Rapi was especially impactful on me! He works combining mapping, technology, activism, and more to empower civil society which is ideally something I would be doing for the rest of my life! One point he made about storytelling spoke to me because he discussed how mapping and passion can be used to promote data/society storytelling. I am passionate about using web development in conjunction with GIS/mapping to work with and within civil society in the (hopefully near) future. I really enjoy the idea of working in tech on societally based projects focusing on people-centered approaches. Just listening to Rapi for an hour has inspired me to go out and seek new ways to hone my passion in this area! Rapi and Lucy overall touched on so many ideas that I support and expressed themselves in ways that really emphasize their passions which I find really valuable. Maintaining a connection with both of these individuals (and Albert!) going forward would be extremely cool!

**Reply by rachan2023** [reply]
That is so awesome that you want to be using mapping, technology, and activism for social change!! Taking a people-centered approach is definitely a big part in giving communities, especially those that are marginalized, a voice. Hearing Rapi speak about how he felt passionate about the Bernie campaign and took action to consolidate all the events that were happening for Bernie solely because he wanted to was really inspiring. Anyways, I can't wait to see the work that you do in the future!!

**Comment by rachan2023** [comment]
I came into the class thinking that web development was about how to find solutions to communities' problems. However, this class changed my perspective about what web development can be used for and how ethics needs to coincide with web development in order to uplift voices in the marginalized communities. I learned to stray away from just finding a solution and to focus on empathizing with my community (and having passion to help my community), understanding the consequences that technology can have on these communities, and finding ways to integrate design justice and data justice to give voice back to my community (instead of using their data, capitalizing on it, and walking away, which institutions and big corporations do so often). I am excited for the future of web development for civil society. Knowing that people are volunteering their time and using open source tools, data justice, and design justice in their web development to help communities is really powerful and inspiring. I saw the passion that all these visiting voices have for their work and it's great knowing that there are people constantly working to help give communities their voice back. With all of these visiting voices, I liked that they saw some kind of problem in society and took an active approach to try to help using web development and/or campaigning. In particular, the visiting voice that left a particular impact on me was Yoh (I love Lucy and Rapi too). Yoh's lecture was really inspiring because he saw what was happening in Japan in the U.S. and wanted to go to Japan to talk to the community that was directly impacted and continued to keep in contact with the people that he interviewed (as opposed to documenting everything and walking away). His lecture really showed me the importance of being passionate about a cause and being accountable for the data/stories that he collected. I definitely would like to keep in contact with Yoh, Lucy, and Rapi and follow the amazing work that they do!!

**Reply by ghost** [reply]
I too entered this class unaware of how damaging it can be to look for solutions and now I know that working with a community to uplift their voices is much more fruitful and empowering than any "solution" would be. I think we both really latched onto the diverse types of passions that the visiting voices brought to this class. Without passion for the subject or work we do, it is hard to find reason in continually putting forth the best efforts we have to offer (at least in my experience). Opposingly, being extremely passionate about something increases the likelihood that you remain diligent in your maintenance of ethics and promotion of data/design justice. This is ultimately why I agree with you that passion is of the utmost importance, especially in the realm of web development for civil society.

**Reply by jason-muljadi** [reply]
To be honest, I didn't even know how web development could really help another community. I always just focused on things at a face-value and never sought to think beyond UI/UX. I was interested in really learning how to design something cool and know that a class that offered a triage of Asian American, Geography, and Computer Science was fascinating. I'm glad that I was able to be a part of this class to really challenge what it means to participate in civil society and how to really improve it. I'm glad to have had a myriad of visiting voices, each with different viewpoints on what it means to map civil society. And I'm also glad to have had this opportunity for collaboration; the greatest impact is the one that is done in collaboration and I am excited to see y'all's work!

**Comment by jason-muljadi** [comment]
This class has really made me think about the tools and services that I use on a regular basis and how these tools can be easily misconstrued by several individuals but can also be used for good. The two visiting voices that really came into my mind was Mariah and Lucy. Mariah really set the foundation of what mapping is all about and how we can visualize it. It was a really powerful introduction to the class and I was blown away by her explanation and the work that she has done. It has shown me how to critically think about cartography and understanding the power of maps. Lucy has shown how technology can be used to benefit civil society, understanding how to work in the tech field while bettering society. I think the unique perspectives of all the visiting voices are wonderful, but these two really stood out to me.

In learning about the community, I hope that we can come together and form a network of individuals to better civil societies, no matter where we are. Although we have had an unprecedented year of turmoil and chaos that I would never wish on anyone, the year has also shown how we can be in community and collaboration through all spheres of the world in order to work towards a common goal to better our society.

**Reply by vcta99** [reply]
Re: collaboration and community in times of crises. I think seeing the work people put in to better civil societies is what gives me sustained hope for the future. The events of the past year have been nothing short but brutal, but I also think they have served to radicalized a lot of folks, and it is encouraging to see the community both growing and also being mindful of the work they must achieve.

**Comment by alizezila** [comment]
Aw, sad to say goodbye to this class :( I think that there is a bright future for web development and civil society, but only if we are reminded of our biases, presumptions, privileges, and our duty to uplift others through our work. This class, people like Albert and all of our visiting voices are so important to the advancement of web development that is accessible, equitable, and open for all to use and benefit from. The work from mega-corporations and institutions that privatizes information only benefits a select few and it is imperative that we work towards a society that pushes for open-source code and development with ethical practices of data collection and usage. The visiting voices that impacted me the most I think were Mariah Tso and Rapi Castillo (the beginning and ends of our visiting voices journey 🥺). Mariah was such an important start to this class for me because it really helped me reframe my understanding of maps and mapmaking, that maps come in many way shapes and forms and with different cultural significance. It taught me that I don't have to make maps with neat lines, legends, or scales and they are still maps that help people navigate life and culture. Rapi's presentation helped me imagine a future for myself with all the different opportunities of map-making that works for the people. Rapi seems super busy, so maybe I won't do as much as him, but his passion and drive for his work is so inspiring! I hope to keep in touch with Albert and my classmates and will definitely be reaching out to those visiting voices! I gained so much insight about data and design justice, accessibility in my web development, and how to make a good survey :) I will keep practicing my HTML and CSS coding with everything I've learned in this class in mind!

**Reply by lvalles10** [reply]
I love what you wrote about working to advance open-source and Mariah Tso's visit! I completely agree that one of the motivations I'm leaving this class with is a drive to push for open-source code and ethical practices of data collection and usage. Although we've learned that open-source code has its drawbacks, by and large I think the lesson to derive from the course is that open-source makes the web development tools necessary to empower civil society accessible. 

I also valued Mariah's presentation. She educated our first breakout group on the importance of data sovereignty and how to create systems of data that operate outside of traditionally oppressive structures of cartography. She also completely blew up the patriarchal and restrictive idea of cartography I entered the class with, which has dramatically changed my perception of how we depict the stories of civil society in our maps.

**Comment by epham36** [comment]
I really appreciated the way this class approached the ethics of web development and civil society! It's a bit disheartening to hear about the ways that civil society has been sidelined in the context of technology development and use, but it is nonetheless exciting to learn so much about tech and web development from an ethical lens, as well as to come into contact with people who are so clearly devoted to this work: the visiting voices, and you, Albert, as well!! Thank you for giving us the skills to be better equipped to be web developers for civil society, as well as skills that can be transferred to support equity in other capacities: through accessible design work, ensuring linguistic accessibility, what it really means to serve the community and being accountable to their needs. Technology and society are mutually constitutive, and while I may not personally be working too much in web development (I may! I don't know what public health needs will be in the future), the awareness of how technology can be a reinforcer or challenger of systemic oppression; and how society can push back against or perpetuate tech determinism -- is something that can be carried out in so many spheres outside of web development. I know there are so many lessons, questions, and skills I am taking from this class. 

Where tech has often been seen as a big bogeyman in my other Asian American studies classes (especially with regards to surveillance, bioinformatics, AI, and policing, etc.), this class also reminded me of the ways in which web development can be used to enact equity, given the proper intent, execution, and collaboration with the community. It also has provided a more place-based and technical skills-focused approach to justice and improving community conditions that I found to be very refreshing. 

I adored Mariah's visit to the class. It really left me floored and so much more excited for mapmaking, geography, and reconsider the way we use and document the physical spaces around us. In a time when my conception of physical space has so often been relegated to just my household and a small radius around that, it was inspiring to think about space, maps, and geography in such a decolonial lens. Regardless, I am still so thankful for all visiting voices and am so glad to have crossed paths with people who are striving to better intertwine civil society and technology. 

I would love to stay connected with all the folks whose lives have crossed mine through this class (visiting voices, classmates, and Albert) -- but I think some visiting voices whose work I am most curious about would be Nina and Adriane, since I am really interested in transportation justice in particular!

**Reply by KenNL42** [reply]
I think I also share similar experience with how my classes usually discuss ethics and technology in terms of what is an ethical line for technology like AI and data justice. I also agree and appreciate how this class takes slightly different approach to web dev and focus on how to empower the society and provide justice and voice to the community. Every visiting voices bring their own experience and approach to share to the class, and it really emphasizes how ethics, technology, and society are intertwined and empower each other.

**Comment by lvalles10** [comment]
My thoughts about the future of web development for civil society are that web development must be a part of a holist solution for civil society’s ills. Web development alone cannot be the single solution to the problems the most marginalized communities face, but web development (including a lens of design and data justice) can be a useful tool in helping mobilize populations and increasing accessibility of movements. 

The visiting voice that left the most impact on me was Lucy! I’m at a point in my undergraduate career where mentorship is essential for success, but I often struggle with how to maintain my mentorship relationships after the initial goal of our relationship is met (ex: getting help applying to a job, programs, etc). Lucy’s talk showed me how to authentically keep in touch with my mentors and even serve as one myself.

I would like to maintain connections with my group members, awesome speakers like Lucy, and Albert going forward! This class has been transformative in shaping my view of civil society, how to solve its problems, and group work in general. I hope to not only keep the connections I’ve made in this class but also retain the knowledge I’ve gained about how to truly fight for social justice in the best interests of civil society’s most marginalized.

**Reply by epham36** [reply]
Yes!! Totally agree with how impactful Lucy's visit was. I feel especially since I am graduating, I've been really trying to seek out guidance from folks who are more experienced than I am in the fields I want to go to, which has been a point of anxiety sometimes. But I really appreciated how Lucy emphasized authentic connection with both mentors and mentees in a way that fosters relationship building and community capacity. Looking back, there are so many mentor/mentee figures that I have, even in an informal capacity that have been so formative for me.

**Comment by rh5140** [comment]
I fully bought into technological solutionism before taking this class, so I’m concerned about web development at large going forward because it doesn’t seem that ethics and community empowerment are emphasized enough in the technological sphere. Like several other students mentioned before, UCLA’s engineering department has just a single wimpy busywork ethics class, and I feel like most free materials for learning web development only teach how to do things and not about what impact development can have.

Mariah had the most impact on me, and her being the first visiting voice let me realize really quickly into the course that I had a lot of unlearning of preconceptions to do. Her presentation was powerful, and I couldn’t stop thinking about it for a while after it. I thoroughly enjoyed this class and want to stay in touch with everyone, as much of a cop-out answer that might sound like. I love reading other people’s thinking cap responses and seeing their labs/projects, every visiting voice has been wonderful, and of course Albert has dedicated so much time to designing and running this class and helping us through it.

**Reply by okcirE** [reply]
I took up a data science bootcamp on udemy recently and I totally understand that there are less emphasis on the humanistic side of technology. The bootcamp taught me to clean up data, make nice visuals, and some statistical testing, but never once have it mentioned any ethical issue such as bias towards the data. I also feel like most of these courses are geared towards teaching you "how to help your company grow" and none of "how can we help people with data science".

**Reply by Wny-Duong** [reply]
Ethics and human-facing technology definitely aren't things that typically get covered within the engineering department unless you really dig in and look for these things (Urban Planning, Digital Humanities, and Civil/Environmental Engineering [specifically CEE 180-187's sequence on transportation] are probably the most directly relevant tech breadth areas relating to civil society). I think this has to boil down with our curriculum wanting us to focus solely on technical skills/information that are most directly applicable to engineering-related research as opposed to applicability towards community concerns and soft skills that would enable you to interact with communities to learn about their concerns. 

I know the case from when I was a freshman and sophomore  was that there really weren't any student organizations that demonstrated interest in tech ethics. More recent stuff like AIRES (AI Robotics Ethics) and ACM Impact's tech ethics lecture series are only recent as of 2019 and 2020 respectively, but I'm hoping those end up making a push for legitimate courses focusing on the topic of tech in the service of civil society in the future (It's how engineering ended up getting blockchain and data science as actual course topics, albeit way later than most other universities I've noticed).

**Comment by ninalong** [comment]
While this was probably not anyone's intention, I do feel a little concerned about the future of web development for civil society. One of the most important lessons I took from this class is the importance of the integration between tech and civil society, particularly in regard to ethics. As someone who comes from the advocacy world, I've always felt a siloing between people who work politics and community development, and those who work tech, and I'm sure those on the tech side would feel the same. While I feel all the members of this class will go into the world equipped with the knowledge and principles to navigate justice in tech, I do worry that few undergo this type of training. 
I found Yoh's presentation really interesting. Drawing in components of environmental research, the ways that marginalized and elderly people are left behind in events of environmental injustice, and community storytelling, Yoh wove a compelling narrative about his experience in Japan. The juxtaposition of the screening in the intimate community space next to that in the vast room of suited businessmen left a lasting impact. 
I would really hope to maintain connections with the folks in this class with more tech-y backgrounds! I've been really awed by their knowledge, and it would be awesome to have connections with expertise in this field particularly for the work I do re: community organizing.

**Reply by alizezila** [reply]
I totally understand your concern for the future, and while I do have hope that web development can be used for good ethical reasons I am also wary of the forces of tech giants and their use of data. So much personal data is sold off nowadays and many people don't know it and have very little say in where their information goes. Though, thats why it is so important to have classes like these so we can keep data justice, accessibility, and ethical web development in mind when we continue our work as professionals!

**Comment by KenNL42** [comment]
Web development and civil society are more relevant than I thought. After learning from this class, I can see how one can empower another, and how important it is to balance between technology/web development and society. In the future, I think web development can empower civil society more easily due to how easy for people to access internet/website through smartphones and social media. Now I can see how much of a power we have to empower society when using web development in the right way.
The visiting voice that left the most impact to me is Rapi. At the beginning of the class, it was quite difficult for me to grasp the idea of using technology to empower society. But after hearing Rapi's experience, it opened my eyes on how much of an impact one person can make to the society via website. I can definitely use the ethics of this class in the future by considering how can a product affects users while also listening to their voice in order to empower the society. I can see how people can overlook the process of making empowered product, but it is quite easy to realize what is needed to empower the society. So I definitely use the ethics from this class to consider how to maximize empowering society.

**Comment by okcirE** [comment]
Web development is a prominent modern tool that most people interact with every day and GIS has proven to be a vital technology during the pandemic. This technology coupled with civil society could bring about change and empower those that have been neglected and marginalized for a long time. Besides the corporate aspect of youtube, I believe that youtube is an awesome example of a site that empowers people's voices, and a platform where people can share knowledge is great. I still vividly remembered the narrating voice of Mariah (the 1st visiting voice), her lecture has helped me connect the dots between our capitalistic past/present and the purpose of technology. And that we must constantly ask hard questions about the intention and who is the technology serving. In general, I think your lectures and the visiting voices have been great and insightful, very assuring that there are also many asian americans tech activist in our space. Thank you Albert for a great quarter and all these puns that I might or might not steal! :)

**Reply by annyrviloria** [reply]
I 100% agree regarding the dots Mariah's presentation helped connect for me. I honestly do not stop thinking about it, and I have continued to challenge myself to unearth my own assumptions about the "power" of tech vis-a-vis the power of us, the creators and users of content, whose decisions will determine how tech is understood and deployed for the greater good in the years to come. Mariah inspires me to be a better ancestor!

**Comment by vcta99** [comment]
It seems almost as if there's an arms race between web development used for exploitation, and that used for civil society. I think the only way to break said "arms race", is if technology is developed in a more empathetic rather than explorative approach. What I mean by that is we need to stop using technology "just to see", without fulling understanding how it may harm others (such as in Coded Bias). Just because we can, doesn't mean we should.

Mariah's presentation regarding critical cartography has stuck with me the most, her presentation almost felt like a one-woman performance piece in how narrative it was. The questioning and re-questioning of definitions regarding data and mapping was very engaging and story-like, and I appreciated it immensely. 

Would love to maintain connections with everyone! If anyone is working on any cool projects over the summer and needs some ux/ui design help (or amateur web dev help), feel free to contact me at vanesschristyta@gmail.com. I know all of y'all will go on to do hella cool things :))

**Comment by Wny-Duong** [comment]
On the future of web development for civil society, I'm honestly pretty optimistic about technology as a whole. Throughout the class, we've definitely been introduced to and demonstrated in the ways that web development might not fully assist communities or realize their needs in a sustainable and humane manner, but I feel that that has more to do with current viewpoints on the role of technology in society versus web development itself. Technology is simply a tool that we use to approach the problems we observe in our daily lives, and to be of use in civil society, we still need to know what to do and how to use such a tool in a proper way. In some forms, technology can serve as a platform for groups of people to use to advocate for change and promote discourse on the problems we see in the world around us. For engineers/developers to achieve societal good going forward, however, I think it's necessary to have a good background in civil society (i.e. taking an interest in the problems around us that either we or more forgotten people in our communities face) or at the very least, developers have to foster a genuine interest in interaction with communities that they're developing for and include those communities as informed partners of the developing process, as opposed to seeing them as a passive audience to web projects.

I thought all the visiting voices were pretty interesting in their own right, but I really enjoyed Yoh, and Nina's presentations the most. Yoh specifically because he covered kind of both kind of extreme ends of technology and society in his work (on looking at the impact of the Fukushima disaster from a more data-oriented perspective before going into further depth by to collecting narratives in-person from the victims of the earthquake), which I found really cool and I think that kind of varied background is probably really good to have going forward when it comes to ethically using technology to address and highlight community concerns and problems. Nina's presentation of sustainable and thoughtful development for civil society was a really good way to ease someone with a strong technical background into developing in a more civic-tech oriented way, which I thought was pretty relevant to me at least, coming from a CS background.

Personally, I'm open to staying in touch with everyone here if anyone has any projects they want to work on over the summer. This class made me a lot more curious about working in the civic tech space so I'm probably going to try and reach out to visiting voices and reach out to Albert as well to see if he has any more advice about how to get involved on that end on a more professional basis.

**Comment by annyrviloria** [comment]
I leave this class with a refreshed sense of hope for more humane web dev. I think there is so much potential to bridge the gap between tech and civil society to advance the greater good and I am now more aware of this thanks to the amazing speakers as well as the lectures from throughout the quarter. As we've discussed before, web dev is only one of the pieces of the puzzle and change cannot depend on it. Yet, web dev is helpful to connect us to other humans working for the causes we care about, interested in joining forces, learning from each other, etc. It's now a matter of putting our ethics above our desire to automate change.


Mariah Tso was the visiting voice that impacted me the most, because she challenged me to situate cartography within the larger historical context it is embedded in. I think I will never forget her presentation, especially when she asked "which of these images is a map" , it was just such a powerful moment, enough to make me think hard and long about my goals as a researcher interested in the connection between urban planning and health.

Going forward, I will continue to check everyone's git hub repositories to get inspiration (and collaborators) for future projects!

**Reply by rh5140** [reply]
I'm glad that this class has given you hope for the future of webdev!

I feel the same about Mariah's presentation — I never would have thought of a bee's waggle dance as a map before. I shared what I learned whichever of my friends would listen to me after her presentation because it was so eye-opening to me. Having to reckon with how some types of maps perpetuate colonialist power structures has made me be more conscious of how things that seem innocuous to me may have a painful history behind them.

I'm planning on keeping up with what our classmates are doing as well after this class!

**Comment by jsxu315** [comment]
I feel more confident in my own ability to create tech to help civil society, but learning about the systems that create barriers to social change made me somewhat pessimistic about practical applications. I think the biggest change is that the idea of ‘using technology to effect social change’ now appears less mysterious and more concrete, mainly because I learned about technological determinism and its flaws — now I see tech as something that can speed up a process or make it more efficient rather than the sole focus of an effort for social change. 

Mariah definitely made the most impact in getting me to think critically about the way I view maps and borders, and it’s something that I think I’ll keep coming back to. Yoh also left a strong impression on me because I saw how staying engaged with a community over the long term helps it and the work that goes into doing so. 

I’d like to stay in touch with everyone because all of their ideas are super interesting; I’ll check everyone’s repos, and I’m also open to helping out with projects.

**Reply by brillianz** [reply]
@jsxu315, I think it's great that you got that takeaway about technological determinism. In an online environment where we are made to believe magnates like Elon Musk and their assumption that technology is our savior, becoming critical about the limits of technology was definitely something that I was put on through this class. How we move beyond these assumptions, and our responsibility to teach others about them, are the questions that we should ask going forward. 

It was also a pleasure being in this class!

**Reply by nnhimichelle** [reply]
I strongly agree with your viewpoints here on the flaws of tech determinism! Now that there is a clearer understanding of technology's effect on social change, it's easier to try and envision a path for improvement of technology as something that could be more efficient to contribute to social change. There's definitely a lot to think about, especially when ethics and data and design justice come to play with technology and how it would effect civil society.

**Comment by brillianz** [comment]
I have many thoughts and takeaways from this class, all of them coming from a place of curiosity and wonder. I wrote something over the weekend that, rather surprisingly, is very close to the prompt that was guided above. I'll place it here, as it brought me much closure to this and some of the broader questions I sought to answer while I was at ucla: 

"I can say with certainty that I have grown even closer through this engagement with my community. At the same time, this project has furthered my understanding of the seamless—often unintentional—relationship that theory carries within my community. I have had the privilege of witnessing and contributing to theoretical dialogues disguised in 'shop talk' arise in different places in my community—whether in barbershops, boxing gyms, park benches, or backyards. Through the skills I gained in both qualitative methods and web development, I am curious about what a mapping project of these conversations through the lens of nemachtili (the spirit of learning in Nahuatl) may signify for our understandings of our own knowledge funds, geographies, and acts of resistance. How might our attitudes towards our cities change themselves towards asset-based and rich? What might such a project reveal of the relationship between knowledge, place, and racialization? Above all, I am excited to serve as a loudspeaker of sorts to amplify the relevant theoretical conversations present on the ground with racialized communities, their desires, and their movement within and beyond the current neoliberal state." 

This class was a warm atmosphere to translate these thoughts and passions through web development, something I never imagined myself exploring. I acknowledge that I had my own preconceived notions about who had access to coding and design justice, and being exposed to multiple speakers who took on these roles made me think otherwise. This was especially the case for Mariah, whose orientation of mapping towards an Indigenous worldview framed what I was learning theoretically to a practical setting. I can say sincerely that this presentation made me feel "seen" and gave me the extra confidence to push through to the end of this course.

**Comment by nnhimichelle** [comment]
This class has made me think a lot about how people can break down the roots of industrial racism and capitalism of the tech industry while working within the tech industry and not succumbing to the rules and other restrictions it may bring. I realized that there is a lot of potential that tech and civil society have to intertwine in creating development in community, and how detailed people have to be when it comes to acknowledging a project's aim and execution. This has overall made me realize to learn to be more careful than I already am, and to always make sure that I take a step back before doing anything. I also learned how to change my personal vocabulary when I speak on certain things and give a more open view rather than using a limited vocabulary that pushes certain narratives among communities. 

Lucy was probably the visiting voice that left the most impact on me, as I want to foster community building through people listening to one another while providing sympathy to each other, as well as opening my eyes on technology for the benefit of civil society. Her activity where we had to talk in partners was particularly interesting, and the rule of avoiding giving advice was definitely something that has made me think over the next couple of days on how I could be a better peer for others to allow room for individual growth and reflection amongst the different paths best for them.

I would love to maintain connections with everyone in the class and see what great things everyone is going to do~! It was amazing seeing everyone's individual words and thoughts, I can't wait to see what the future holds for each and every one of you :)

**Reply by jsxu315** [reply]
I really like what you said about Lucy's activity! It made me think about the importance of connecting with project team members, which I hadn't considered much before, as well as the community. I think that your point about taking a step back is also super important, because it helps us reduce potential biases in the tech we create and keep learning how to better help civil society.

**Reply by ninalong** [reply]
I'm definitely interested in how folks like you will be able to break down the issues in tech from within while resisting profit motivations. We will always be engaging with technology, and it will remain an essential tool for civil society; thus those working within tech will have important roles in ensuring its ethical use. I'm excited for what you'll be able to do with the tools and principles we've learned in this class!!

**Comment by skarakas1** [comment]
I think that web development will be necessary for social change. As big tech continues to try to monopolize information technology, it is up to civil society to push back to create a digital and physical world that is more democratic and equitable. All of the visiting voices were great, but Mariah, Yoh, Rapi, and Carolanne were highlights for me and have helped me imagine more creative and fulfilling possibilities for my own future. Of course you are also a huge inspiration Albert! This is one of the best classes I've taken for a few reasons including the amount of different guest lecturers and the focus on praxis. I think it's awesome that we are creating functional and hopefully useful applications with thought and ethical consideration behind them. Great work Albert! You have convinced me further of the necessity of grassroots community empowerment in a field largely dominated by big data.

**Reply by latishasumardy** [reply]
I really agree with you, and I definitely feel like it is important for us to use our knowledge and technical abilities to fight against corporations who are trying to use it for unethical things. I also loved all of the visiting voices, and Rapi and Yoh were definitely also highlights for me! Finally, I agree with you 100% about Albert, and I am really glad I decided to take this class this quarter as I have learned so much about how we can use applications to empower communities from him.

**Reply by avaitt** [reply]
Loved your take on web development here. I really like how you mentioned Yoh's talk. I think his methodological process of documenting victim's stories and experiences helped create a sense of humanity to the horrible tragedy that was Fukushima. I also think Albert's class was awesome this quarter! I loved each lecture and getting an idea of how social empowerment works hand-in-hand with data.

**Comment by latishasumardy** [comment]
I think that web development is definitely something that can be utilized more in order to create social change. There are so many tools available to us online, and being able to utilize them correctly to do activism is something that can promote more and more social change, as long as we keep in mind what our initial goal is. My favorite visiting voice was Rapi, because I thought that his transition into becoming more active in advocating for social change was really relatable, as I am also planning to go into software development. I hope that I can also become more involved in activism and will one day be able to utilize my skills to create social change in my community. As for connections, I would really love to keep in touch with Lucy, because she's so cool, Rapi, and of course Albert, because Albert is the one who has really gotten me more interested and excited about being able to apply my skills to advocate for change in communities.

**Comment by avaitt** [comment]
Each of the visiting voices conveyed something awe-inspiring and eye-opening. I truly enjoyed learning about each of their backgrounds and the work they have done to forward social justice in their respective disciplines. Rapi's talk was especially interesting to me because of how he meshed javascript and mapping technologies in the context of social justice and activism. GIS to me is a truly mind-blowing technology and I love seeing how experts utilize it to create truly amazing. One aspect of Rapi's talk that was especially interesting to me was his discussion about using his skill-sets to help his own community and campaign for people who bring real change. It never occurred to me that social justice can sometimes be close to home. Listening to Rapi has led me to think about how I can create real change within my own community by meshing social justice with design. I'd love to maintain a connection with Rapi and Albert as they both participate in truly inspiring work!
